compilation errors on 'cargo install gurk'

```
cargo install gurk

error[E0277]: the trait bound `std::vec::Vec<tui::layout::Constraint>: std::convert::From<[tui::layout::Constraint; 2]>` is not satisfied
  --> /home/oren/.cargo/registry/src/github.com-1ecc6299db9ec823/gurk-0.1.0/src/ui.rs:18:10
   |
18 |         .constraints([Constraint::Ratio(1, 4), Constraint::Ratio(3, 4)])
   |          ^^^^^^^^^^^ the trait `std::convert::From<[tui::layout::Constraint; 2]>` is not implemented for `std::vec::Vec<tui::layout::Constraint>`
   |
   = help: the following implementations were found:
             <std::vec::Vec<T> as std::convert::From<&[T]>>
             <std::vec::Vec<T> as std::convert::From<&mut [T]>>
             <std::vec::Vec<T> as std::convert::From<std::borrow::Cow<'a, [T]>>>
             <std::vec::Vec<T> as std::convert::From<std::boxed::Box<[T]>>>
           and 5 others
   = note: required because of the requirements on the impl of `std::convert::Into<std::vec::Vec<tui::layout::Constraint>>` for `[tui::layout::Constraint; 2]`

error[E0277]: the trait bound `std::vec::Vec<tui::layout::Constraint>: std::convert::From<[tui::layout::Constraint; 2]>` is not satisfied
  --> /home/oren/.cargo/registry/src/github.com-1ecc6299db9ec823/gurk-0.1.0/src/ui.rs:81:10
   |
81 |         .constraints([
   |          ^^^^^^^^^^^ the trait `std::convert::From<[tui::layout::Constraint; 2]>` is not implemented for `std::vec::Vec<tui::layout::Constraint>`
   |
   = help: the following implementations were found:
             <std::vec::Vec<T> as std::convert::From<&[T]>>
             <std::vec::Vec<T> as std::convert::From<&mut [T]>>
             <std::vec::Vec<T> as std::convert::From<std::borrow::Cow<'a, [T]>>>
             <std::vec::Vec<T> as std::convert::From<std::boxed::Box<[T]>>>
           and 5 others
   = note: required because of the requirements on the impl of `std::convert::Into<std::vec::Vec<tui::layout::Constraint>>` for `[tui::layout::Constraint; 2]`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0277`.
error: failed to compile `gurk v0.1.0`, intermediate artifacts can be found at `/tmp/cargo-installPopXGn`

Caused by:
  could not compile `gurk`.

To learn more, run the command again with --verbose.
```

More info:
```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial
```

---

```
$ uname -a
Linux or

--- a/src/ui.rs
+++ b/src/ui.rs
@@ -15,7 +15,7 @@ use std::path::PathBuf;
 
 pub fn draw<B: Backend>(f: &mut Frame<B>, app: &mut App) {
     let chunks = Layout::default()
-        .constraints([Constraint::Ratio(1, 4), Constraint::Ratio(3, 4)])
+        .constraints([Constraint::Ratio(1, 4), Constraint::Ratio(3, 4)].as_ref())
         .direction(Direction::Horizontal)
         .split(f.size());
 
@@ -78,10 +78,13 @@ fn draw_chat<B: Backend>(f: &mut Frame<B>, app: &mut App, area: Rect) {
     };
 
     let chunks = Layout::default()
-        .constraints([
-            Constraint::Min(0),
-            Constraint::Length(num_input_lines as u16 + 2 + extra_cursor_line),
-        ])
+        .constraints(
+            [
+                Constraint::Min(0),
+                Constraint::Length(num_input_lines as u16 + 2 + extra_cursor_line),
+            ]
+            .as_ref(),
+        )
         .direction(Direction::Vertical)
         .split(area);
 

